### DISCUSSION TITLE
name 'traceback' is not defined

### DISCUSSION THREAD

**Discussion by fq393** [open-webui/open-webui#7712] [Issues]
# Bug Report

![image](https://github.com/user-attachments/assets/fdb2796c-6a67-41c5-90c0-2787d305e08b)

---

## Installation Method

Docker deployment

## Environment

- **Open WebUI Version:** v0.4.8

- **Ollama:** 0.5.1

- **Operating System:** rockey8
- **Browser (if applicable):** Chrome

**Comment by fq393** [comment]
using this plugin： https://openwebui.com/f/tokyohouseparty/artifacts_v2/
But the report is wrong： name 'traceback' is not defined
I didn't see the specific reason in the log either. Help me

**Comment by fq393** [comment]
There is no problem in removing this plug-in. I tried it. Artifacts is the native support?

**Comment by Spring2022abcjk** [comment]
I used the same plug. I had the same problem
